vendor/k8s.io/kubernetes/test/e2e/common/downwardapi_volume.go

```diff
@@ -273,7 +273,7 @@ func downwardAPIVolumePodForModeTest(name, filePath string, itemMode, defaultMod
 			VolumeMounts: []v1.VolumeMount{
 				{
 					Name:      "podinfo",
-					MountPath: "/etc",
+					MountPath: "/etc/podinfo",
```

Comment:
thanks for fixing this! I noticed this while working on a flake for this test and was surprised when it overwrote /etc :P